app/source/components/notifications/notifications.jsx

```diff
@@ -59,6 +59,14 @@ class Notifications extends React.Component {
     });
   }
 
+  componentWillUnmount() {
+    if (!this.unsubscribe) {
```

Comment:
Will this ever happen?

Reply:
Probably not, but I'm so very defensive 😅.

Reply:
This is your chance to live a little on the wild side!